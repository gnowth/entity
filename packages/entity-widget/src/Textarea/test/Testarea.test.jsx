import 'jest-styled-components';
import renderer from 'react-test-renderer';
import React from 'react';

import WidgetTestarea from '..';

it('renders correctly', () => {
  const tree = renderer.create(
    <WidgetTestarea
      name="test"
      onChange={() => undefined}
    />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
