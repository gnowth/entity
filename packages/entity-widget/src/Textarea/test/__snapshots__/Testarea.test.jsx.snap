// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  width: 100%;
}

<textarea
  className="c0"
  name="test"
  namespace="component_widgetTextarea"
  onChange={[Function]}
  value=""
  variant="standard"
/>
`;
